# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scroll-to-text-fragment/force-load-at-top.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/scroll-to-text-fragment/force-load-at-top.html.ini
--- a/testing/web-platform/meta/scroll-to-text-fragment/force-load-at-top.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/scroll-to-text-fragment/force-load-at-top.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,12 +1,16 @@
 [force-load-at-top.html]
-  expected: TIMEOUT
+  expected:
+    if os == "mac": OK
+    TIMEOUT
   [force-load-at-top must block scroll on load from element fragment.]
     expected:
       if os == "android": ["FAIL", "TIMEOUT"]
       FAIL
 
   [no-force-load-at-top must not block scroll on load from history scroll restoration.]
-    expected: NOTRUN
+    expected:
+      if os == "mac": PASS
+      NOTRUN
 
   [no-force-load-at-top must not block scroll on load from text fragment.]
     expected:
@@ -15,14 +19,14 @@
 
   [no-force-load-at-top must not block scroll on load from text fragment with element fallback.]
     expected:
+      if os == "win": ["TIMEOUT", "FAIL", "NOTRUN"]
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["TIMEOUT", "NOTRUN", "FAIL"]
-      if os == "mac": ["TIMEOUT", "FAIL", "NOTRUN"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [no-force-load-at-top must not block scroll on load from element fragment.]
     expected:
       if os == "linux": ["TIMEOUT", "NOTRUN"]
+      if os == "mac": ["PASS", "TIMEOUT", "NOTRUN"]
       [NOTRUN, TIMEOUT]
 
   [force-load-at-top must block scroll on load from history scroll restoration.]